### PR TITLE
Update enablemfa.md

### PR DESCRIPTION
**Pull request by AhsenBaig**
Updated documentation for setting up Google Authenticator.

## Description:
- **What's changed?** 
Updated documentation for setting up Google Authenticator.

### CODE REVIEW

**Comment by AhsenBaig** [comment]
The verbiage should match what is in the other docs: 
https://help.okta.com/oie/en-us/Content/Topics/identity-engine/authenticators/google-authenticator.htm

`
1. In the Admin Console, go to Security > Authenticators.
2. On the Setup tab, click Add Authenticator.
3. Click Add on the Google Authenticator tile and again in the details screen.
4. Enroll Google Authenticator in a multifactor policy.
`

File: packages/@okta/vuepress-site/docs/guides/mfa/main/ga/enablemfa.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-1. From the Admin Console, select **Security** from the main menu and then **Multifactor**.
-2. On the **Factor Types** tab, select **Google Authenticator** from the left navigation.
-3. Click the **Inactive** drop-down box, and then select **Activate**.
+1. From the Admin Console, select **Security** from the main menu and then **Authenticators**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/main/ga/enablemfa.md:1]:
```suggestion
1. In the Admin Console, go to **Security** > **Authenticators**.
```

File: packages/@okta/vuepress-site/docs/guides/mfa/main/ga/enablemfa.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+2. Click the **Add Authenticator** button, select **Google Authenticator** (OTP).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/main/ga/enablemfa.md:2]:
```suggestion
2. Click **Add Authenticator**, and then click **Add** on the Google Authenticator tile.
```

File: packages/@okta/vuepress-site/docs/guides/mfa/main/ga/enablemfa.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
...
+3. Click the Add button associated with the service.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/main/ga/enablemfa.md:3]:
```suggestion
3. Click **Add** to add the service. 
```

**Review by susanharper-okta — COMMENTED** [review]
edits

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**